src/styles/index.css

```diff
@@ -0,0 +1,12 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Should not be applied to * , As guided by Paul.

Reply:
Recommend way of CSS reset.